recipes-devtools/upm/upm_0.7.1.bb

```diff
@@ -16,16 +16,14 @@ inherit distutils-base pkgconfig python-dir cmake
 CFLAGS_append_edison = " -msse3 -mfpmath=sse"
 
 FILES_${PN}-doc += " ${datadir}/upm/examples/"
-RDEPENDS_${PN} += " mraa"
+RDEPENDS_${PN} += " mraa mraa-java"
 
 # override this in local.conf to get a subset of bindings.
 # BINDINGS_pn-upm="python"
 # will result in only the python bindings being built/packaged.
 
-#Disable the python package generation until we can get it building consistently on all platforms/architectures.
-# note, we can rely on BINDINGS_${PN}-mraa if we want these controlled by 1 variable
-#BINDINGS ?= "python nodejs java"
-BINDINGS ?= "nodejs java"
+# Build all language bindings by default
+BINDINGS ?= "python nodejs java"
```

Comment:
@tripzero the recipe rename pulls the 0.7.1 tagged version. The new UPM version (0.7.1) defines which python version (2 vs 3) to use as a hint to find_package PythonLibs  

I will rebase my PR to NOT add the mraa-java rdepends since this seems to be blocking the python binding fix from getting merged.

I'm not sure about the note on line 26.